bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/models/v2/PageImpl.java

```diff
@@ -148,6 +150,8 @@
     private String[] clientLibCategoriesJsHead;
 
     private List<HtmlPageItem> htmlPageItems;
+    
+    public static final String CANONICAL_URL = "canonical";
```

Comment:
Constants should be defined in the model interface and follow the name convention there.

Reply:
This has been taken care

Reply:
@vladbailescu Did you get a chance to review the changes? These changes taken care upon your review comments?